esp/services/WsDeploy/WsDeployService.cpp

```diff
@@ -5305,6 +5305,19 @@ bool CWsDeployFileInfo::handleRows(IEspContext &context, IEspHandleRowsRequest &
         const char* compType = pRow->queryProp("@compType");
         xpath.clear().appendf("Override[@component='%s'][@instance='%s']", compType, rowName);
       }
+      else if (!strcmp(rowType, "CustomBindingParameter"))
```

Comment:
is it worthwhile to do a case insensitive compare? Or do we control the CustomBindingParameter label?